IBPSA/Resources/C-Sources/cryptographicsHash.c

```diff
@@ -312,3 +313,5 @@ const char* cryptographicsHash(const char* str)
 
   return hexresult;
 }
+
+#endif
```

Comment:
@AntoineGautier looks like this needs a newline?

Reply:
@Mathadon I did not know that. Is is related to [https://stackoverflow.com/questions/72271/no-newline-at-end-of-file-compiler-warning](https://stackoverflow.com/questions/72271/no-newline-at-end-of-file-compiler-warning)?
(I added it to fileWriterStructure.c as well.)

Reply:
yup, it's a bit of a detail but a nice convention to uphold when coding